README.md

```diff
@@ -1,38 +1,74 @@
----
-tags: ternary, ||=, operators
-language: ruby
-resources: 4
----
-
 # Operations
 
 ## Objectives
 
 Get familiar with: 
 
+* Learn about the ternary operator
+* Review using comparison operators and boolean operators
 * passing arguments to methods
-* passing multiple arguments to a method
 * manipulating arguments within the methods
 * returning the desired values from a method
 
-## Instructions
+## Reviewing Operations
+
+In this lab, we'll be combining our knowledge of operators (like comparison and boolean operators) with our new understanding of flow control using `if/else` statements. 
+
+This lab will require you to use the comparison operators (>, <, etc.) and boolean operators you learned in the previous unit. You can review them [here](https://learn.co/tracks/web-development-with-ruby-on-rails/ruby/booleans-and-math/boolean-operators). 
+
+There's one more operator that we'll need to understand before we proceed with this lab:
+
+## The Ternary Operator
+
+The ternary operator is another type of comparison operator that is used in the context of `if/else` statements. An `if/else` statement without the ternary operator might look something like this: 
+
+```ruby
+age = 1
+
+if age < 2
+	"baby"
+else
+	"not a baby"
+end 
+```
+
+Remember that if the condition after the `if` returns `true`, the code underneath the `if` will execute. Otherwise, the code underneath the `else` will execute. In the above example, the value of the `age` variable is in fact less than 2, so the above code will return `true`. 
 
-**Note: Remember to delete comments, otherwise the spec won't pass!**
+Let's take a look at the same code using the ternary operator: 
 
-To complete this lab, you will be adding code to `lib/operations.rb`. The tests are written in order so get the first to pass before continuing on to the second, etc.
+```ruby
+age = 1
+
+age < 2 ? "baby" : "not a baby"
+```
+
+How does this work? In the above statement, the code before the `?` is evaluated as a boolean expression. If it returns true, the code on the left side of the `:` will run, otherwise the code on the right of the `:` will run. 
+
+Think about it like this: 
+
+```ruby
+boolean_expression ? I_happen_if_boolean_expression_is_true : I_happen_if_boolean_is_false
+```
+## Why Use the Ternary Operator?
+
+The ternary operator has a few things going for it. First of all, as Rubyists, we like to think about design. Is our code needlessly complex? Are our methods too long? Is our code self-explanatory––i.e., can someone reading it easily understand what it does? These are good questions to keep in mind as you learn to build more and more complicated programs. In fact, Ruby is a language that lends itself particularly well elegant design. There are many design patterns that you'll learn about later on in this course, but for now, just keep the above questions in mind. 
```

Comment:
Leet's give a more concrete opinion on when to use the ternary operator. In my opinion it should never be used....but I know many people disagree with me. How about only if the condition is `incredibly simple` or something?